docs/operations/hotplug_interfaces.md

```diff
@@ -100,6 +100,34 @@ You can now check the VMI status for the presence of this new interface:
 kubectl get vmi vmi-fedora -ojsonpath="{ @.status.interfaces }"
 ```
 
+### Migration based hotplug
+In case your cluster doesn't meet the requirements, it's possible to hotplug an interface by migrating the VM.
+The actual attachment wont take place immediately, the new interface will be ready once the migration is completed.
```

Comment:
```suggestion
The actual attachment won't take place immediately, and the new interface will be available in the guest once the migration is completed.
```

I think we should be a bit more specific above, thus I've rephrased a bit the second premise.

Reply:
Done